exchanges/bitstamp/bitstamp.go

```diff
@@ -663,5 +670,23 @@ func (b *Bitstamp) SendAuthenticatedHTTPRequest(path string, v2 bool, values url
 
 	encodedValues := values.Encode()
 	readerValues := strings.NewReader(encodedValues)
-	return b.SendPayload("POST", path, headers, readerValues, result, true, b.Verbose)
+
+	interim := json.RawMessage{}
+
+	errCap := struct {
+		Error string `json:"error"`
+	}{}
+
+	err := b.SendPayload("POST", path, headers, readerValues, &interim, true, b.Verbose)
+	if err != nil {
+		return err
+	}
+
+	if err := common.JSONDecode(interim, &errCap); err == nil {
```

Comment:
Does this mean that when JSONDecode errors into err, it is uncaught?

Reply:
Yeah if it errors we do not need to handle that because of the secondary unmarshal.